xmipp

```diff
@@ -24,15 +24,12 @@
 # ***************************************************************************/
 
 
-import distutils.spawn
-import glob
-import json
 import os
 import re
 import shutil
 import sys
-import subprocess
 from datetime import datetime
+from scripts import *
```

Comment:
I would import only the neccesary like import scripts as spts. and then would call spts.function. But it's ok from my side